README.md

```diff
@@ -1,19 +1,37 @@
 cloud-dns
 =========
 
-Role to manage Route53 registers.
+What this role do?
+  * Compare all local dns records wich aws and generate a var wich
+    * changed records
+    * Diferences between AWS and local records
+  * Notifies slack or hipchat of changes
+  * Apply chaged records
+  * Notifies fails or sucessfuly rebuilds
 
 Requirements
 ------------
 
-ansible 2.4 or newer.
+ansible 2.3 or newer.
 
 Variables
 --------------
 ```yaml
-zone: domain.com # zone to be managed by Role.
-env: all # Can be any subdirectory in the root of the domain directory
-global_vars: { GLOBAL_DNS_DEFAULT_TTL: 60 } # Var dict for substitution
+# route53_check vars
```

Comment:
It's a good practice to use var names with an prefix to identifier the role.

Reply:
:arrow_up: @edsoncsouza please, add this on issues as a TODO task.